elftools/dwarf/dwarfinfo.py

```diff
@@ -91,6 +92,9 @@ def __init__(self,
                 DebugSectionDescriptor for a section. Pass None for sections
                 that don't exist. These arguments are best given with
                 keyword syntax.
+            
+            max_cu_cache_size:
+                Enforces a limit on CU cache size, For unlimitted  cache size set to -1
```

Comment:
The size limit is on the number of entries/CUs? The comment should say this in more detail.